### DISCUSSION TITLE
导出为图片时，可否选择以PPT形式导出。

### DISCUSSION THREAD

**Discussion by whileGreatHair** [tw93/MiaoYan#251] [Ideas]
目前导出的图片为正常md渲染后的形式导出的。
需求：以妙言PPT每一页为一张图导出。这样能够方便的将图片放入PPT以演示。毕竟很多时候不太好直接用妙言演示，或者有时需要将PPT分享给他人。

**Comment by tw93** [comment]
试试 导出妙言 PPT PDF 满足需求不？

**Comment by whileGreatHair** [comment]
试过啦，首先，妙言PPT PDF导出的内容排版没法用，因为会出现一页PPT的内容，出现在PDF的两页内容里（被分开了），第二点是PDF的格式也不太方便做成真正的PPT。如果是把每页妙言PPT直接输出为独立图片的话，只需要把图片分别放到空白的PPT里，就可以直接当成PPT使用啦。（感谢您
